SwaprHQ/swapr-dapp
# [SWA-67][FEAT] - Add native $USDC on Arbitrum

## Fixes: [SWA-67](https://linear.app/swaprdev/issue/SWA-67/update-usdc-token-in-arbitrum)

# Description

* Add $USDC native token in Arbitrum

![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/13aa9885-7cbc-4818-9588-79fda71f0ed5)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet
5) $USDC and $USDC.e tokens should be among the common tokens suggested for swaps on Arbitrum One
